cpu/saml21/ldscripts/saml21.ld

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright (C) 2019 ML!PA Consulting GmbH
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @addtogroup      cpu_saml21
+ * @{
+ *
+ * @file
+ * @brief           Memory definitions for Atmel SAML21
```

Comment:
```suggestion
 * @brief           Memory definitions for Microchip SAML21
```

Reply:
Well even my shiny new same54 still has Atmel printed on it, but names are sound and smoke 😉

Reply:
True ! But my contacts at Microchip kindly ask me to rename everything 😉